addons/sale_mrp/models/sale.py

```diff
@@ -14,25 +14,29 @@ class SaleOrder(models.Model):
 
     @api.depends('procurement_group_id.stock_move_ids.created_production_id.procurement_group_id.mrp_production_ids')
     def _compute_mrp_production_count(self):
+        data = self.env['procurement.group'].read_group([('sale_id', 'in', self.ids), ('mrp_production_ids', '!=', False)], ['id'],['sale_id'])
```

Comment:
add a space between `,['sale_id']` or the fw-bot in master will be red :/

Reply:
ok, done